docs/src/pages/components/pickers/pickers.md

```diff
@@ -20,24 +20,53 @@ packageName: '@material-ui/lab'
 
 {{"demo": "pages/components/pickers/MaterialUIPickers.js"}}
 
+### Setup
+
+You need to provide a date-library that is used by the pickers by setting the `dateAdapter` to an adapter of your choosing.
+
+We currently support 4 different date-libraries:
+
+- [date-fns](https://date-fns.org/)
+- [Day.js](https://day.js.org/)
+- [Luxon](https://moment.github.io/luxon/#/)
+- [Moment.js](https://momentjs.com/)
+
+First you have to install the adapter package for the date-library you want to use:
+
+```sh
+// date-fns
+npm install @date-io/date-fns
+// or for Day.js
+npm install -s @date-io/dayjs
+// or for Luxon
+npm install -s @date-io/luxon
+// or for Moment.js
+npm install @date-io/moment
+```
+
+Then you have to set the `dateAdapter` prop of the `LocalizationProvider` accordingly:
+
+```js
+// date-fns
+import DateAdapter from '@material-ui/lab/AdapterDateFns';
+// or for Day.js
+import DateAdapter from '@material-ui/lab/AdapterDayjs';
+// or for Luxon
+import DateAdapter from '@material-ui/lab/AdapterLuxon';
+// or for Moment.js
+import DateAdapter from '@material-ui/lab/AdapterMoment';
+
+function App({ children }) {
+  return (
+    <LocalizationProvider dateAdapter={DateAdapter}>{children}</LocalizationProvider>
+  );
+}
+```
+
```

Comment:
We could provide a recommendation for those unfamiliar with differences between different libraries (as suggested in https://github.com/mui-org/material-ui/issues/23522#issuecomment-644218429)

Reply:
I'm usually very cautious with these recommendations since we're not experience with these libraries usually. I'd rather curate a list of articles comparing these libraries. Similar to how you should style components in your application.